lib/main/pages/app_pages.dart

```diff
@@ -92,13 +90,6 @@ class AppPages {
                 contact_view.loadLibrary,
                 () => contact_view.ContactView()),
             binding: ContactBindings()),
-        GetPage(
-            name: AppRoutes.identityCreator,
-            opaque: false,
-            page: () => DeferredWidget(
-                identity_creator.loadLibrary,
-                () => identity_creator.IdentityCreatorView()),
-            binding: IdentityCreatorBindings()),
```

Comment:
I don't think it is a good idea, please care to MOBILE, WEB, TABLET the same. @dab246 please care this case

Reply:
- I've checked it doesn't affect any platform.
- Previously we moved to the new full screen screen on mobile devices so we need to use `Page` to `navigate` to it. But according to the new design, `@Khai` has used `bottomSheetDialog` on mobile to display the view, so there is no need for `Page` anymore.